src/components/chat-view-container/chat-view.tsx

```diff
@@ -240,10 +243,28 @@ export class ChatView extends React.Component<Properties, State> {
             )}
             {this.props.messages.length > 0 && <Waypoint onEnter={this.props.onFetchMore} />}
             {this.props.messages.length > 0 && this.renderMessages()}
-            {!this.props.hasLoadedMessages && <ChatSkeleton conversationId={this.props.id} />}
+            {this.props.messagesFetchStatus === MessagesFetchState.IN_PROGRESS && (
+              <ChatSkeleton conversationId={this.props.id} />
+            )}
+
             <div ref={this.bottomRef} />
           </div>
         </InvertedScroll>
+
+        {this.props.messagesFetchStatus === MessagesFetchState.FAILED && (
```

Comment:
I think maybe we need to be careful here. What if we've already previously loaded the messages and then we open a conversation? We show the existing messages but we also then fetch in the background to kinda ensure we get the latest stuff. But, if that one fails I don't think we want to clear the entire conversation. We might not even want to show the actual error....maybe we don't care...or maybe we want a more subtle indicator.

Reply:
> But, if that one fails I don't think we want to clear the entire conversation. We might not even want to show the actual error....maybe we don't care...or maybe we want a more subtle indicator.

I don't know...what the right answer is here. For me personally i agree that we shouldn't clear the entire conversation. BUT i think not showing "any" error is a more buggy state..since we know that the user has missed some latest messages..but the user doesn't know that! 

I think in this case we should show an error at the bottom, with keeping the previous conversation as is. And example message could be "Some messages failed to load. Try Reloading".

Reply:
In this PR i am doing the same in theory (keeping the previous conversation as is). But ..just not able to reproduce the state so that i actually see it on my local (i.e allowing the first message fetch, and then failing the next ones for the same channel). Will try more.

Reply:
Yep, that's the kind of thing Eth should have input on.

Reply:
looks like this now

<img width="1496" alt="image" src="https://github.com/zer0-os/zOS/assets/33264364/0f9bf864-d6db-4ffd-8815-198d45dda726">

<img width="457" alt="image" src="https://github.com/zer0-os/zOS/assets/33264364/6ce4e1fb-b126-4972-a632-4e17ecdfcd62">